iml-wasm-components/iml-action-dropdown/src/lib.rs

```diff
@@ -188,6 +194,15 @@ fn open_class(open: &bool) -> &str {
     }
 }
 
+/// Sends the custom event up to the window, carrying with it the data. Should we move this into its own module?
+fn dispatch_custom_event(_type: &str, data: &JsValue) {
```

Comment:
Instead of taking a `JsValue` and forcing the user to serialize it should take anything that implements `Serialize` and it unsized.